libseccomp/src/lib.rs

```diff
@@ -45,6 +45,10 @@
 //!     Ok(())
 //! }
 //! ```
+//!
+//! # Features
+//!
+//! - `scmpsyscall_new`: Allow creating of `ScmpSyscall` in a `const`-context.
```

Comment:
> because `rustc` doesn't support the platforms ("targets") currently.

Which effectively means that you can not use libseccomp-rs on these architectures.

Reply:
Oh, you're right. I didn't have to need mentioned it here.